extensions/wikia/Email/Email.i18n.php

```diff
@@ -3,6 +3,7 @@
 
 $messages['en'] = array(
 	'emailext-desc' => 'An extension to handle delivering email',
+	'emailext-anonymous-editor' => 'A Wikia fan',
```

Comment:
Can the capitalization issue in https://wikia-inc.atlassian.net/browse/SOC-641 be handled as part of your changes here?